subprojects/core/src/main/java/org/gradle/internal/progress/BuildProgressLogger.java

```diff
@@ -56,23 +58,36 @@ public void projectsLoaded(int totalProjects) {
     public void beforeEvaluate(String projectPath) {}
 
     public void afterEvaluate(String projectPath) {
-        if (!taskGraphPopulated) {
+        if (!rootTaskGraphPopulated) {
             buildProgress.progress("", false);
         }
     }
 
     public void graphPopulated(int totalTasks) {
-        taskGraphPopulated = true;
+        rootTaskGraphPopulated = true;
         buildProgress.completed();
         buildProgress = loggerProvider.start(EXECUTION_PHASE_DESCRIPTION, EXECUTION_PHASE_SHORT_DESCRIPTION, totalTasks);
     }
 
+    public void nestedTaskGraphPopulated(int totalTasks) {
```

Comment:
I see, thanks for clarifying. Wondering if @big-guy and @bigdaz already have an established terminology to describe this.